src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/config/AuditConfig.java

```diff
@@ -17,14 +17,15 @@
  */
 public class AuditConfig {
 
+    public static final List<String> DEFAULT_IGNORED_USERS = Collections.singletonList("kibanaserver");
```

Comment:
Is it possible for DEFAULT_IGNORED_USERS to be different for AuditConfig and ComplianceConfig?

Reply:
It is the same as kibanaserver is the only special user whose requests we want to ignore. This change has been moved to #484